Userland/Libraries/LibGfx/Painter.cpp

```diff
@@ -1248,7 +1244,7 @@ void Painter::draw_glyph_or_emoji(IntPoint const& point, u32 code_point, Font co
 
     // Perhaps it's an emoji?
     auto* emoji = Emoji::emoji_for_code_point(code_point);
-    if (emoji == nullptr) {
+    if (!emoji) {
```

Comment:
This change seems unnecessary. In fact, clang-tidy recommends explicit nullptr checks.

Reply:
Fixed.